src/components/Signin.jsx

```diff
@@ -0,0 +1,265 @@
+                                    <h5 className="text-danger">Oops! There was an error.</h5>
+                                )}
+                            </div> */}
+                            <div className="form-group text-center">
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignIn
+                              </Button>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                width="60%"
+                                marginLeft="5px"
+                              >
+                                Forgot your password?
+                              </Button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <Seperator >
+                                <h2><span>OR</span></h2>
+                              </Seperator>
+                              <DisplayForLinks >
+                                <SocialLinks >
+                                  <li>
+                                    <SocialMediaLink  color ="white" bgColor="#3B5998" href="#" className="btn">
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </SocialMediaLink>
+                                  </li>
+                                  <li >
+                                    <SocialMediaLink bgColor="black" color="white" hoverBgColor="darkgrey" href="#" className=" btn">
+                                      <i className="fa fa-github "></i> Signin with Github
+                                    </SocialMediaLink>
+                                  </li>
+                                  <li >
+                                    <SocialMediaLink color="blue" bgColor="lightblue" href="#" className=" btn">
+                                      <LinkedinIcon className="fa fa-linkedin-square " ></LinkedinIcon><LinkedinLinkContent >Signin with Linkedin</LinkedinLinkContent>
```

Comment:
Why aren't you using same thing with LinkedIn?

Reply:
becuase the defualt color and font size of this icon is different from others, thats y i needed to adjust it.

Reply:
It is really looking odd, use the same tags. Create styled component for 'i' if needed. But all links should look same. Refer to some site with such authentication window.